### ISSUE TITLE
fail import ssl and maybe other modules

### ISSUE THREAD

**Issue by jgirardet** [beeware/briefcase#435]
Hi,

edit: Maybe duplicate of #420

I'm on kubuntu 19.10
I initialy post in #46  where some libs where not included but I think I 'm wrong, the case is different. Here the lib is present but not found :

latest briecase.
just add `import ssl" to the default app
Following error :
```
[mycartable] Built linux/MyCartable-0.0.1-x86_64.AppImage

[mycartable] Starting app...

Traceback (most recent call last):
  File "/tmp/.mount_MyCarthLt0Ty/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/tmp/.mount_MyCarthLt0Ty/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmp/.mount_MyCarthLt0Ty/usr/app/mycartable/__main__.py", line 2, in <module>
    import ssl
  File "/tmp/.mount_MyCarthLt0Ty/usr/lib/python3.7/ssl.py", line 98, in <module>
    import _ssl             # if we can't import it, let the error propagate
ImportError: libssl.so.1.0.0: cannot open shared object file: No such file or directory
```

libssl is inside the bunddle :
```
jimmy@cacahuete:/tmp/.mount_MyCart8cSbpd/usr/lib$ ls -l
total 29019
-rw-r--r--  1 root root    71504 juin  27 09:21 libbz2.so.1.0
-rw-r--r--  1 root root  2439344 juin  27 09:21 libcrypto.so.1.0.0
-rw-r--r--  1 root root    42336 juin  27 09:21 libcrypt.so.1
-rw-r--r--  1 root root    34896 juin  27 09:21 libffi.so.6
-rw-r--r--  1 root root    13704 juin  27 09:21 libgdbm_compat.so.3
-rw-r--r--  1 root root    26288 juin  27 09:21 libgdbm.so.3
-rw-r--r--  1 root root   142480 juin  27 09:21 liblzma.so.5
-rw-r--r--  1 root root    98400 juin  27 09:21 libnsl.so.1
-r-xr-xr-x  1 root root 25525426 mai   27 08:40 libpython3.7m.a
-rw-r--r--  1 root root   879144 juin  27 09:21 libsqlite3.so.0
-rw-r--r--  1 root root   444344 juin  27 09:21 libssl.so.1.0.0
drwxr-xr-x 27 root root        0 mai   27 08:41 python3.7
```
Jimmy

**Comment by freakboy3742** [comment]
Thanks for the report.

To help us work out what has gone on here:
1. What version of Briefcase are you using?
2. Was the app built using Docker (the default with Briefcase 0.3.1)? If not, did you build a custom support package?

**Comment by jgirardet** [comment]
briefcase 0.3.1
Yes the app was build with docker

**Comment by freakboy3742** [comment]
On digging into #420, this is almost certainly a duplicate. The issue will be that the module hasn't been relocated correctly, and there's a system reference to libssl lingering. I've been able to reproduce your problem, and the fix that I have for #420 fixes this problem too.
